src/app/(private)/admin-registration-applications/[id]/page.tsx

```diff
@@ -1,8 +1,156 @@
+import { Input } from '@/components/ui/input'
+import {
+  Select,
+  SelectTrigger,
+  SelectValue,
+  SelectContent,
+  SelectItem,
+} from '@/components/ui/select'
+import { AdminRegistrationApplication } from '../types'
+import { formatDate } from '@/app/hocs/formatDate'
+import Status from '@/modules/adminRegistrationApplications/components/status'
+
+const Page = ({ params }: { params: { slug: string; id: string } }) => {
+  const [
+    adminRegistrationApplicationDetails,
+    setAdminRegistrationApplicationDetails,
+  ] = useState<AdminRegistrationApplication | null>(null)
+  const [loading, setLoading] = useState(true)
+  const [error, setError] = useState<any>(null)
+
+  useEffect(() => {
+    const fetchDetails = () => {
+      getAdminRegistrationApplication(params.id)
+        .then((response) => {
+          console.log(response)
+          if (response.data.isSuccess) {
+            console.log(response.data.response)
+            setAdminRegistrationApplicationDetails(response.data.response)
+          } else {
+            setError(new Error('Failed to fetch data'))
+          }
+          setLoading(false)
+        })
+        .catch((err) => {
+          setError(err)
+          setLoading(false)
+        })
+    }
+    fetchDetails()
+  }, [params.id])
+
+  const renderInput = (label: string, value: string | number | null) => (
+    <div className="w-full">
+      <label className="block text-sm font-medium text-gray-500">{label}</label>
+      <Input value={value ?? ''} disabled className="mt-1 block w-full" />
+    </div>
+  )
+
+  const renderPhoneInput = (countryCode: string, lineNumber: string) => {
+    if (countryCode && lineNumber) {
+      return (
+        <div className="w-full">
+          <label className="block text-sm font-medium text-gray-500">
+            Phone Number
+          </label>
+          <Input
+            value={`(+${countryCode}) ${lineNumber}`}
+            disabled
+            className="mt-1 block w-full"
+          />
+        </div>
+      )
+    } else {
+      return (
+        <div className="w-full">
+          <label className="block text-sm font-medium text-gray-500">
+            Phone Number
+          </label>
+          <Input value="" disabled className="mt-1 block w-full" />
+        </div>
+      )
+    }
+  }
+
+  return (
+    <div className="form-container p-6 bg-white dark:bg-gray-800 rounded-md shadow-md text-black dark:text-white">
+      {loading && <div>Loading...</div>}
+      {error && <div>Error: {error}</div>}
+      {!loading && !error && adminRegistrationApplicationDetails && (
+        <form className="space-y-4">
+          <div className="mb-6">
+            <h1 className="text-xl font-bold mb-1">
+              Institution ID:{' '}
+              {adminRegistrationApplicationDetails.institution?.id}
+            </h1>
+            <div className="text-m text-gray-250 space-x-6">
```

Comment:
Bu kısım için task içine açıklama eklenmiştir. Sayfa başlığının listelemedeki gibi olması gerekmektedir. ve id burada gösterilmeyecektir.